jigglypuffRL/deeprl/ddpg/ddpg.py

```diff
@@ -79,6 +80,7 @@ def __init__(
         pretrained=False,
         save_name=None,
         save_version=None,
+        save_model=None,
```

Comment:
Lesser the number of user arguments the better. Can't we use just two (`save_model` and `save_interval`)? Instead of explicitly providing `save_name`, the files can be saved as `<algo>_<env_name>_<run_number>/log-<iteration_number>` and `save_version` is probably not necessary right now. What do you think @Sharad24 @Het-Shah ?

Reply:
I agree with @ajaysub110

Reply:
Can you elaborate what you mean by run_number? Also, I can't figure out a way to send env_name without making env_name another argument to the function.

Reply:
Usually in RL, one has to do multiple runs because results vary a lot.

For getting the env_name you can do `env.unwrapped.spec.id`

Reply:
Ok thanks. So multiple runs meaning the number of times the .py file is run, or the number of times the learn method is used after declaring the algo object?

Reply:
Yes, the number of times the `.py` files is run. Sorry for the late reply.